### ISSUE TITLE
Support for Kotlin 1.9.0

### ISSUE THREAD

**Issue by Sardorbekcyber** [mars885/hilt-binder#48]
After updating Kotlin from "1.8.22" to "1.9.0" and 
KSP "1.8.22-1.0.11" to "1.9.0-1.0.11" all bindings failing 

Error Hilt is giving is you need to annotate with @Provides or @Binds

**Comment by kamilargent** [comment]
@mars885 any update here?

**Comment by mars885** [comment]
Yes, I'm going to release a new version soon that fixes this issue. Stay tuned.

**Comment by hamid97m** [comment]
Same issue here; we will be grateful if you release the new version as soon as possible.

**Comment by hamidHeli** [comment]
> Yes, I'm going to release a new version soon that fixes this issue. Stay tuned.

Have you any plans for the new release? we should migrate to KSP in Kotlin 2 in our project

**Comment by hamid97m** [comment]
> Yes, I'm going to release a new version soon that fixes this issue. Stay tuned.

Seriously, have you any plans for this migration? :D

**Comment by mars885** [comment]
Is this still relevant?

I'm in a process of updating dependencies and releasing a new version for the Kotlin 1.9.24. However, I haven't done any fixes to get the lib working again. The only issue I found is that some versions of the Hilt was giving me compatibility problems where my bindings did not work. I just followed this [thread on StackOverflow](https://stackoverflow.com/a/68319048) and resolved the issue by upgrading Hilt to 2.48.
